src/charactersheet/utilities/notifications.js

```diff
@@ -131,6 +131,18 @@ export var Notifications = {
         changed: new Signal()
     },
 
+    encounter: {
+        added: new Signal(),
+        changed: new Signal(),
+        deleted: new Signal()
+    },
+
+    encounterimage: {
```

Comment:
looks like we're pretty consistent in the naming of notifications with `camelCase`. The new ones that were added in this PR should be updated.

Reply:
They can't be. These add/change/delete notifications have to be all lower (at least for now).

Currently, the subscription names are [automatically generated from the Model class name](https://github.com/adventurerscodex/adventurerscodex/blob/master/src/charactersheet/viewmodels/abstract/tabular-view-model.js#L135) which is just the lowercase name of the model.

```
// Sample from the abstract VMs.
setUpSubscriptions() {
    const modelName = this.modelClass().name.toLowerCase();  // <-----
    this.subscriptions.push(Notifications[modelName].added.add(this.addToList));
    this.subscriptions.push(Notifications[modelName].changed.add(this.replaceInList));
    this.subscriptions.push(Notifications[modelName].deleted.add(this.removeFromList));
}
```

Reply:
hm, interesting design choice. In that case, it would be good to at least write a remark as to why the change in naming occurred here.

Reply:
These are all from the v2 changes. Not from mine. I can add some docs to the Notifications tho.